queries.sql

```diff
@@ -6,3 +6,36 @@ SELECT name, escape_attempts from animals WHERE weight_kg > 10.5;
 SELECT * from animals WHERE neutered = 't';
 SELECT * from animals WHERE name <> 'Gabumon';
 SELECT * from animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+BEGIN;
```

Comment:
- [ ] Great job so far... like the previous reviewer noted, kindly add comments of the different transactions you did in the query.sql file. this will not only help a reviewer to know what you did but will help you as well if you need to use any of them. 😄